manager/api/rest-impl/src/main/java/io/apiman/manager/api/schema/SchemaRewriterService.java

```diff
@@ -124,27 +126,40 @@ public String rewrite(ProviderContext providerCtx, InputStream is, ApiDefinition
 
     private static class SchemaCacheKey {
         private final long apiVersionId;
-        private final ApiDefinitionType apiDefinitionType;
```

Comment:
Is the `ApiDefinitionType` not also needed?

Reply:
Not saying it _is_, but just want to understand why it was removed rather than 3-part composite key.

Reply:
I wrote this a while ago, so may need to refresh my understanding of it.

Reply:
Okay, makes sense.